holy-books.org

```diff
@@ -625,50 +625,72 @@ E.g. Quran:7:157 results in text “Quran 7:157” with a tooltip showing the ve
+                            color size
+                            (holy-books-quran chapter verse)
+                            (if no-info-p
+                                ""
+                              (format
+                               (concat
+                                "<small>"
+                                "<a href="
+                                "\"https://quran.com/chapter_info/%s?local=en\">"
+                                "Quran %s:%s, %s"
+                                "</a>"
+                                "</small>")
+                               chapter
+                               chapter
+                               verse
+                               (cl-getf (cl-getf holy-books-quran-cache chapter)
+                                        :name)))))
+                   ((eq 'md backend)
+                    (format "\n> %s\n>\n> %s"
+                            (holy-books-quran chapter verse)
+                            (if no-info-p
+                                ""
+                              (format "[Quran %s:%s   %s](https://quran.com/chapter_info/%s)"
+                                      chapter
+                                      verse
+                                      (cl-getf (cl-getf holy-books-quran-cache chapter)
+                                               :name)
+                                      chapter)))))))
+
+ :face '(:foreground "green" :weight bold))
 
 
 ;; Quran:chapter:verse|color|size|no-info-p
 (org-link-set-parameters
-  "Quran"
-  :follow (lambda (_) nil)
-  :export (lambda (label _ __)
-            (-let* (((chapter:verse _ __ ___) (s-split "|" label))
-                    ((chapter verse) (s-split ":" chapter:verse)))
-              (format "<abbr class=\"tooltip\"
+ "Quran"
+ :follow (lambda (_) nil)
+ :export (lambda (label _ backend)
+           (-let* (((chapter:verse _ __ ___) (s-split "|" label))
+                   ((chapter verse) (s-split ":" chapter:verse)))
+             (cond
+              ((eq 'html backend)
+               (format "<abbr class=\"tooltip\"
                              title=\"﴾<em> %s</em>﴿ <br><br> %s <br><br> %s\">
                           Quran %s:%s
                        </abbr>&emsp13;"
-                      (holy-books-quran chapter verse)
-                      (cl-getf (cl-getf holy-books-quran-cache chapter) :name)
-                      (format "https://quran.com/%s" chapter)
-                      chapter verse)))
-  :face '(:foreground "green" :weight bold))
+                       (holy-books-quran chapter verse)
+                       (cl-getf (cl-getf holy-books-quran-cache chapter) :name)
+                       (format "https://quran.com/%s" chapter)
+                       chapter verse))
+              ((eq 'md backend)
+               (format "[Quran %s:%s](%s \"﴾%s﴾\n\n %s\")" chapter verse
+                       (format "https://quran.com/%s" chapter)
```

Comment:
This and the next 3 lines are repeated above, maybe a `let` clause to factor out the similarities ;-)

Reply:
OH! I didn't realize this part got pushed as well. I was planning another PR to add markdown support. 

But, I'll fix this as well  :)

Reply:
I'm looking forward to it! Keep up the great work :-)